# store the data for plot

float64 data_1
float64 data_2
float64 data_3
float64 data_4
float64 data_5
float64 data_6
float64 data_7
float64 data_8
float64 data_9
float64 data_10
float64 data_11
float64 data_12
float64 data_13
float64 data_14
float64 data_15
float64 data_16
float64 data_17
float64 data_18
float64 data_19
float64 data_20
float64 data_21
float64 data_22